### PR TITLE
Update Emoji Completions

### PR DESCRIPTION
**Pull request by oxinabox**
We are missing like half a decade of new Emoji 😨 
Thats like 356 new emoji

I initially just used the newest list but I found that it removed some, I think basically just ones that were legacy of Emoji in unicode from v1.0 which often don't have nice renderings, like ㊙ and ☕️ (both of which render nice for me, but i have seen them render e.g. in black and white in some fonts)

So then i changed to merging the new list on top of the old list.
Problem here is some have been renamed.
When 🥚 was added it became `:egg:`,
and the current `:egg` 🍳 was renamed to `:fried_egg:`
We might prefer this, as these names are better, but idk might be too breaking.
(Through probably not, since REPL interface is excluded from our semver guarantee)

For now though i have set it to make sure nothing ever is removed or renamed by merging older on top of newer

### CODE REVIEW

File: stdlib/REPL/src/emoji_symbols.jl [hunk 1]
```diff
@@ -650,41 +933,70 @@ const emoji_symbols = Dict(
...
+    "\\:shushing_face:" => "🤫",
     "\\:signal_strength:" => "📶",
     "\\:six_pointed_star:" => "🔯",
+    "\\:skateboard:" => "🛹",
     "\\:ski:" => "🎿",
-    "\\:skin-tone-2:" => "\U1f3fb",
-    "\\:skin-tone-3:" => "\U1f3fc",
-    "\\:skin-tone-4:" => "\U1f3fd",
-    "\\:skin-tone-5:" => "\U1f3fe",
-    "\\:skin-tone-6:" => "\U1f3ff",
```
Comment (oxinabox) [inline stdlib/REPL/src/emoji_symbols.jl:683]: These are the same values.
I am not sure what changed that changed how they are made into a string.

**Comment by stevengj** [comment]
Traditionally, you should have waited until 2021-04-01 to post this PR.

**Review by stevengj — APPROVED** [review]

**Comment by oxinabox** [comment]
@stevengj do you have an opinion on the 🥚vs 🍳 question?

Right now we avoid change, at the cost of having no way to enter 🥚

**Comment by stevengj** [comment]
I would just rename 🍳 `:fried_egg:`.   We're not constrained by backwards compatibility here.

For that matter, I think it would be fine to replace the whole old list with the new list.

**Comment by musm** [comment]
@oxinabox any chance you plan on updating this PR based on 

> For that matter, I think it would be fine to replace the whole old list with the new list.

**Comment by oxinabox** [comment]
Rebased  and changed to still be merging but new should overwrite old.
Minimized breaking peoples workflow, still gives us 100% of the new ones.

**Review by stevengj — APPROVED** [review]

**Comment by musm** [comment]
Looks like an unrelated  error in `testset errorshow:`

**Pull request merged**